portfolio/src/main/java/com/google/sps/servlets/StoreCard.java

```diff
@@ -16,17 +18,22 @@
 import com.google.maps.GeoApiContext;
 import com.google.maps.PhotoRequest;
 import com.google.maps.ImageResult;
+import com.google.maps.model.OpeningHours;
+import com.google.maps.model.LatLng;
 
 public class StoreCard {
     String placeId ="";
     String name = "";
     float rating = 0.0f;
     String photoString;
+    Boolean openNow;
```

Comment:
Use primitive `boolean` vs. boxed `Boolean` and make this 
```
boolean openNow = false;
```

This will save you a line (see comment below).